tox.ini

```diff
@@ -46,11 +53,8 @@ deps =
     nose
     coverage
     psycopg2
-    MySQL-python
-
-# we separate coverage into its own testenv because a) "last run wins" wrt
-# cobertura jenkins reporting and b) pypy and jython can't handle any
-# combination of versions of coverage and nosexcover that i can find.
-# coverage <3.4 is required by nosexcover 1.0.4.
+    mysqlclient
 
-# DB='pgsql' DBURI='postgresql+psycopg2://user:password@host/dbname' tox -e py32
+[tox:travis]
```

Comment:
this is required for invoking the cover and lint targets in addition to the database/python version build matrix.
I've elected to only run them on the 2.7 and 3.5 targets and keep coverage to 2.7 (as it was before limited to use python 2.7)